test_container/Dockerfile

```diff
@@ -3,11 +3,11 @@ MAINTAINER Xin Liang <XLiang@suse.com>
 
 CMD ["/usr/lib/systemd/systemd", "--system"]
 
-RUN zypper refresh && \
-    zypper -n install systemd \
-        make autoconf automake vim which libxslt-tools mailx iproute2 iputils bzip2 openssh tar file glibc-locale-base firewalld libopenssl1_1 dos2unix iptables cpio \
-        python3 python3-pip python3-lxml python3-python-dateutil python3-setuptools python3-PyYAML python3-curses python3-behave \
-        csync2 libglue-devel corosync corosync-qdevice pacemaker booth corosync-qnetd
+RUN zypper refresh
+RUN zypper -n install systemd
+RUN zypper -n install make autoconf automake vim which libxslt-tools mailx iproute2 iputils bzip2 openssh tar file glibc-locale-base firewalld libopenssl1_1 dos2unix iptables cpio
+RUN zypper -n install python3 python3-pip python3-lxml python3-python-dateutil python3-setuptools python3-PyYAML python3-curses python3-behave
+RUN zypper -n install csync2 libglue-devel corosync corosync-qdevice pacemaker booth corosync-qnetd
```

Comment:
Splitting it into multiple steps result in more layers in container images and slows down the container. And running refresh before installing package help to ensure all the packages is latest.

Reply:
Changed